Dockerfile

```diff
@@ -120,41 +113,28 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         libmcrypt-dev \
         libncurses5-dev \
         libnss3 \
-        libpng12-dev \
+        libpng-dev \
         libreadline6-dev \
         librsvg2-bin \
         libsm6 \
         libsqlite3-dev \
         libssl-dev \
         libtiff5-dev \
+        libtool \
         libwebp-dev \
-        libwebp5 \
+        libwebp6 \
         libxml2-dev \
         libxrender1 \
         libxslt-dev \
         libxss1 \
         libxtst6 \
+        libvips-dev \
         libyaml-dev \
         mercurial \
         nasm \
         openjdk-8-jdk \
         optipng \
-        php5.6 \
-        php5.6-xml \
-        php5.6-mbstring \
-        php5.6-gd \
-        php5.6-sqlite3 \
-        php5.6-curl \
-        php5.6-zip \
-        php5.6-intl \
-        php7.2 \
-        php7.2-xml \
-        php7.2-mbstring \
-        php7.2-gd \
-        php7.2-sqlite3 \
-        php7.2-curl \
-        php7.2-zip \
-        php7.2-intl \
```

Comment:
Nice to see all these deprecated versions getting attention!

Looks like the install script, which runs as the `buildbot` user, doesn't auto-install any PHP versions, just selects one of the preinstalled versions based on https://github.com/netlify/build-image/blob/xenial/run-build-functions.sh#L374-L388

Since this PR removes all preinstalled versions, what do you think about researching some installation method that works without root so that users can install the version they need at runtime? I wonder if homebrew would be a good alternative.

Reply:
Ah, I missed that we still have that one :+1: 

I don't know enough about PHP to know if we have a lot of users on old versions and what the upgrade path would be like for them. I do think providing some auto-install at runtime would be cool but we'll know more about whether PHP versions are a pain point once we start beta testing the image, so this seems fine for now.

Reply:
➕1 to iterate as we get feedback for this image and if needed we can consider manually installing or as you suggested doing something at runtime.